### ISSUE TITLE
Acquire Few Images from the validation set

### ISSUE THREAD

**Issue by LiuZichuan** [ZHKKKe/MODNet#92]
Hi, 

Can I have some image samples from the val set of PPM-100? The quality of my re-implemented model still have gap when compared with the official pretrained. I want to see how my model perform on the validation set of PPM-100.

My email: zliu106@e.ntu.edu.sg

Thanks

**Comment by ZHKKKe** [comment]
Hi, thanks for your attention.

I am sorry that I cannot release PPM-100 before I get permission from my supervisor.
But the validation set should be released soon.

**Comment by ddiddu** [comment]
Thanks for the great repository! 
It genuinely helped me alot. 

Will I be able to know when will be the validation set released?

Thanks

**Comment by ZHKKKe** [comment]
@ddiddu 
In our schedule, it will be released next month. Thanks.

**Comment by ddiddu** [comment]
@ZHKKKe Thanks for the reply!
